proposals/util_test.go

```diff
@@ -59,7 +59,7 @@ func TestComputeWeightPerEligibility(t *testing.T) {
 func TestComputeWeightPerEligibility_EmptyRefBallotID(t *testing.T) {
 	types.SetLayersPerEpoch(layersPerEpoch)
 	signer, err := signing.NewEdSigner(
-		signing.WithKeyFromRand(rand.New(rand.NewSource(1001))),
+		signing.WithKeyFromRand(rand.New(rand.NewSource(1000))),
```

Comment:
Why have you changed the source?

Reply:
The test uses signatures to determine eligibility. Since I changed which data goes into a signature the previous identities would not be eligible any more. I changed which identities are being used in the test to some that are eligible for the given active set.